ambry-network/src/main/java/com/github/ambry/network/LocalRequestResponseChannel.java

```diff
@@ -45,13 +46,18 @@ public void sendRequest(NetworkRequest request) {
   }
 
   @Override
-  public NetworkRequest receiveRequest() throws InterruptedException {
+  public NetworkRequestBundle receiveRequest() throws InterruptedException {
```

Comment:
is the local channel in use ?

Reply:
Yeah, local channel was actually used for Cloud requests. I will remove it in a separate pr.